projects/lib-project/src/lib/dock/components/tasks/tasks.component.ts

```diff
@@ -295,12 +295,25 @@ export class TasksComponent implements OnInit, OnDestroy {
     }
   }
 
-  adjustValue(event: any): void {
-    const inputValue = parseInt(event.target.value, this.tasksData.minEvidences.validators.max);
+  adjustValue(event: any, task:any): void {
+    const inputValue = parseInt(event.target.value, this.tasksData.minEvidences.validators.max); // Convert the input value to a number
     const min = this.tasksData.minEvidences.validators.min;
     const max = this.tasksData.minEvidences.validators.max;
-    const clampedValue = Math.min(Math.max(inputValue, min), max);
-    event.target.value = clampedValue.toString();
+    let adjustedValue = inputValue;
```

Comment:
why adjustedValue here ?
we use InputValue itself for comparision and to holding the value